swift-day-3.txt

```diff
@@ -0,0 +1,146 @@
+1)
+There are 2 ways to declare functions inside class or struct i.e using 'class' and 'static' keyword.
+
+static func f1(){}
+class func f2(){}
+
+
+-Static functions can be invoked by the className itself.Object creation is not required.
+->static functions can not be inherited.
+class Vehicle{
+  static func drive(){
+  
+  }
+}
+we can access static function drive as Vehicle.drive()
+-Static functions can not be overridden like class functions.
+-In case of static functions,when we access any static function,entire class gets loaded into memory.
+
+-Class functions are not instance methods but can be invoked using className and can be overridden by subclasses 
+-Class functions also can be invoked without object creation
+class Vehicle{
+    class func drive(){
+        println("vehicle driving")
+    }
+}
+class Bike: Vehicle{
+    override class func drive(){
+        println("Bike Driving")
+    }
+}
+
+These class method can be called directly using classnames as Bike.drive() and Vehicle.drive()
+----------------------------------------------------------------------------------------------------------------------------------------------------------
+2)
+func displayOrders(orders:[Menu]){
+  for order in orders{
+   switch order{
+       case let .Burger(patties):
+            print("burgers with \(patties) patties")
+       case let .Fries(size):
+            print("\(size) Fries")
+       case .Meal:
+            print("Meal")
+            print("Coke")
```

Comment:
we have `print("Coke")` here and at L46, which seems like violation of DRY principle. Is there a way to remove this duplication?